README.md

```diff
@@ -41,6 +43,10 @@ This is the list of Functions available in this repo:
 - [ ] Voicemail
 - [ ] Inbound calls for SIP registration
 - [ ] Outbound calls for SIP registration
+- [ ] Generate Video access token
+- [ ] Generate Sync access token
+- [ ] Generate Chat access token
+- [ ] Conference line
 - [ ] Translate webhook from `application/x-www-form-urlencoded` into `application/json` and forward on to another service
```

Comment:
Since these functions have now been written (see above), they don't need to be in the Todo section.